jenkins/toolbox-run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
ditto, convoluted way instead of `$PWD` or `pwd`

Reply:
This was done this way so the script could be run from anywhere in the repo. If we were to use $PWD it would only work as expected if run from the right dir.